BigQuery/tests/System/ManageDatasetsTest.php

```diff
@@ -110,6 +115,33 @@ public function testUpdateDataset()
         $this->assertEquals($metadata['friendlyName'], $info['friendlyName']);
     }
 
+    public function testSetDatasetDefaultEncryption()
+    {
+        $encryption = new KeyManager(
+            json_decode(file_get_contents(getenv('GOOGLE_CLOUD_PHP_TESTS_KEY_PATH')), true),
+            'bq-145106218721@bigquery-encryption.iam.gserviceaccount.com'
```

Comment:
Would you be able to use `ENCRYPTION_SERVICE_ACCOUNT_EMAIL_TEMPLATE` here?

Reply:
fixed, thanks.